postmaster/templates/admins.html

```diff
@@ -43,6 +45,31 @@ <h2 class="textHeadingManage">PostMaster Administrators</h2>
     <ul class="pagination hidden" id="pagination"></ul>
 </div>
 
+<!-- 2 Factor Modal -->
+<div class="modal fade" id="twoFAModal" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Setup 2 Factor</h4>
+            </div>
+            <div class="modal-body">
+                <div id="qrcode"></div>
+                <p>Scan QR code with an App like Authy or Google Authenticator</p>
+                <div class="input-group">
```

Comment:
This comes up looking goofy on my end:
![image](https://cloud.githubusercontent.com/assets/11711106/18228168/d05d82d2-720e-11e6-9af7-ae33fc210be8.png)